src/Microsoft.CodeAnalysis.Analyzers/CSharp/MetaAnalyzers/CSharpReportDiagnosticAnalyzer.cs

```diff
@@ -33,6 +33,23 @@ protected override IEnumerable<SyntaxNode> GetArgumentExpressions(InvocationExpr
 
                 return null;
             }
+
+            protected override SyntaxNode GetPropertyGetterBlockSyntax(SyntaxNode declaringSyntaxRefNode)
```

Comment:
The name is a bit weird now.

Reply:
I'd really like to switch this to an IOperation analyzer soonish to avoid more syntax related gymnastics.